Runtime/Scripts/Plugins/AnimationPointer/MaterialPropertiesRemapper.cs

```diff
@@ -259,7 +259,7 @@ public DefaultMaterialPropertiesRemapper()
         {
             var baseColor = new MaterialPointerPropertyMap(MaterialPointerPropertyMap.PropertyTypeOption.SRGBColor)
             {
-                PropertyNames = new[] { "_Color", "_BaseColor", "_BaseColorFactor", "baseColorFactor" },
+                PropertyNames = new[] { "_Tint", "_Color", "_BaseColor", "_BaseColorFactor", "baseColorFactor" },
```

Comment:
i just changed the order of _Color and _BaseColor on the base branch. Please test that this is even still required